app/routes/admin/events/list.js

```diff
@@ -125,7 +125,7 @@ export default class extends Route.extend(EmberTableRouteMixin) {
       'page[number]' : params.page || 1
     };
     // add condition for default sorting the table by date
-    if (!params.sort_by && !params.sort_dir) {
+    if (!params.sort_by || !params.sort_dir) {
```

Comment:
Now if any of them is not present, both will get replaced

Reply:
i am really sorry to make so many pull reqests. do i need to write them in two different if conditions?

Reply:
See just above your addition how default page size and page number are handled

Reply:
But the code seems to work perfectly fine...i am not able to get the problem in code.

Reply:
Now if any of them is not present, both will get replaced

Reply:
> seems to work perfectly fine

That's the problem. It seems to work fine. But it doesn't

Reply:
that is what we want to do?
arrange the the events in order of date if no sorting parameter is there.